src/bin-support.js

```diff
@@ -37,6 +38,55 @@ async function runTransform(binRoot, transformName, args, extensions = DEFAULT_E
   }
 }
 
+async function runTemplateTransform(
+  binRoot,
+  transformName,
+  args,
+  extensions = DEFAULT_TEMPLATE_EXTENSIONS
+) {
+  const globby = require('globby');
+  const execa = require('execa');
+  const chalk = require('chalk');
+  const path = require('path');
+  const { parseTransformArgs } = require('./options-support');
+
+  let { paths, options } = parseTransformArgs(args);
+
+  try {
+    let foundPaths = await globby(paths, {
+      expandDirectories: { extensions: extensions.split(',') },
+    });
```

Comment:
Hm, so first I just copied it from the JS util. It does work with passing `paths` directly, tested that, but then we cannot pass custom extensions, as we are currently able to do. They are hardcoded in `ember-template-recast`: https://github.com/ember-template-lint/ember-template-recast/blob/master/src/runner.js#L184.

The JS util also allowed passing custom extensions as the optional last argument, although that was not used on the `bin/cli.js` generator. But honestly I don't really care, we can certainly remove this argument for the hbs util, and just pass to `execa` what we got!?

Reply:
> But honestly I don't really care, we can certainly remove this argument for the hbs util, and just pass to execa what we got!?

Yeah, lets do that. I plan to iterate over in ember-template-recast, and having one less layer on top of it would make that easier...

Reply:
Done.